typescript/cli/src/tests/commands/submit.ts

```diff
@@ -0,0 +1,24 @@
+import { $ } from 'zx';
+
+import { localTestRunCmdPrefix } from './helpers.js';
+
+export async function hyperlaneSubmit({
+  transactions,
+  registry,
+  privateKey,
+  hypKey,
+}: {
+  transactions: string;
+  registry: string;
+  privateKey?: string;
+  hypKey?: string;
+}) {
+  return $`${
+    hypKey ? ['HYP_KEY=' + hypKey] : []
+  } ${localTestRunCmdPrefix()} hyperlane submit \
+        --registry ${registry} \
+        --transactions ${transactions} \
+        ${privateKey ? ['--key', privateKey] : []} \
+        --verbosity debug \
+        --yes`;
```

Comment:
I feel like i'm seeing double:
- https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6703/files#diff-6b1f089e410312c29fab7695259c283d20b2937be025da5de65baa0ed7682af2
- https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6703/files#diff-fcaa80bdec8d7bb5e2067bdef56f8441642bf3f45474be518dc13af0a76c61c6

Reply:
ha, copied over and forgot